externals/llvm-external-projects/torch-mlir-dialects/lib/Conversion/TcpToLinalg/Elementwise.cpp

```diff
@@ -106,6 +106,40 @@ createLinalgPayloadForElementwiseOp(Operation *op,
     return {b.create<arith::DivFOp>(loc, one, sum)};
   }
 
+  if (isa<MinOp>(op)) {
+    if (elemType.isa<mlir::FloatType>()) {
+      return {b.create<arith::MinFOp>(loc, payloadArgs[0], payloadArgs[1])};
+    }else if (elemType.isa<mlir::IntegerType>()){
+      mlir::IntegerType intElemType = elemType.dyn_cast<mlir::IntegerType>();
+      if (intElemType) {
+        if (intElemType.isSigned()) {
```

Comment:
TCP only has SignlessIntegers: https://github.com/llvm/torch-mlir/blob/mlir-tcp/externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpBase.td#L35

We can't figure out the signedness from the element type in TCP. So, we will have to encode the signedness on the op itself, which is very similar to what llvm does. This is the same for all of core MLIR.

Due to this, we will have to add 3 different min / max ops, one for each of Float, Signed Int and Unsigned Int types, similar to how `tcp.divf` is implemented (`tcp.divsi` and `tcp.divui` are still TBD).